avm/res/cdn/profile/tests/e2e/afd.premium/main.test.bicep

```
@@ -9,7 +9,7 @@ metadata description = 'This instance deploys the module as Azure Front Door Pre
 
 @description('Optional. The name of the resource group to deploy for testing purposes.')
 @maxLength(90)
-param resourceGroupName string = 'dep-${namePrefix}-cdn.profiles-${serviceShort}-rg'
+param resourceGroupName string = 'dep-${namePrefix}-afd.premium-${serviceShort}-rg'
```

```suggestion
param resourceGroupName string = 'dep-${namePrefix}-cdn.profiles-${serviceShort}-rg'
```
Should remain as it was to make it obvious to which module the test belongs. Conflcits are avoided using the `serviceShort`